testing/asserts_test.ts

```diff
@@ -860,6 +968,26 @@ Deno.test("AssertRejectsWithErrorCallback", async () => {
   );
 });
 
+Deno.test("assertThrows returns caught error", () => {
```

Comment:
Can you also add similar test cases where the thrown values are not errors, but `string`s, `number`s, etc?

Reply:
Sure, will add more test cases.